raven/utils/compat.py

```diff
@@ -46,3 +46,17 @@
     from urllib import parse as _urlparse  # NOQA
 
 urlparse = _urlparse
+
+
+def check_threads():
+    try:
+        from uwsgi import opt
+    except ImportError:
+        return
+
+    if str(opt.get('enable-threads', '0')).lower() in ('false', 'off', 'no', '0'):
```

Comment:
plz comment with related uwsgi source code: https://github.com/unbit/uwsgi/blob/master/core/uwsgi.c#L4010-L4019 in case these values change in the future.